server/initializers/constants.ts

```diff
@@ -451,6 +461,8 @@ const VIDEO_PRIVACIES: { [ id in VideoPrivacy ]: string } = {
   [VideoPrivacy.INTERNAL]: 'Internal'
 }
 
+const VIDEO_CHANNEL_MAX_SYNC = 3
```

Comment:
I'm not sure to understand the purpose of this constant?

Reply:
It's passed to youtube-dl / yt-dlp. It is the number of the latest of videos of the remote channel to check and maybe to upload. Without that limitation, there are performance issues (it can take very long).

The naming is bad. However I miss an idea for a better one.

Reply:
You can promote this constant as an admin setting (not mandatory to provide it on the web interface) :)

Reply:
Done in 37082c1d3e3040be8580fb5b53d92d32e3e9e626